test/browser/browser_asrouter_bookmarkpanel.js

```diff
@@ -16,11 +16,6 @@ add_task(async function test_fxa_message_shown() {
   const testURL = "data:text/plain,test cfr fxa bookmark panel message";
   const browser = gBrowser.selectedBrowser;
 
-  await SpecialPowers.pushPrefEnv({
```

Comment:
This wasn't needed anymore. For a short while we had `devtoolsEnabled` set in our `browser.ini` but we reverted that after some issues.